ci/browser-tests/cypress/integration/activity.spec.js

```diff
@@ -1,15 +1,29 @@
 describe("Activity view", function() {
-    before(() => {
-        cy.dummyLogin("benjamin")
-        cy.setup("task", {"project-name": "ACTIVITY TESTING",
-            "activity": "preliminary-design"})
+    beforeEach(() => {
+        cy.visit("")
+        cy.get("#password-textfield").type(Cypress.env("SITE_PASSWORD"));
```

Comment:
why not use the cy.dummyLogin helper?

Reply:
cy.dummyLogin helper doesn't open quick project search...